recipe/build.sh

```diff
@@ -68,18 +78,21 @@ function source_build
 
       export CPATH=$BUILD_PREFIX/include
       export LIBRARY_PATH=$BUILD_PREFIX/lib
-      export CC=${CC_FOR_BUILD}
-      export CXX=${CXX_FOR_BUILD}
-      export CPP=${CXX_FOR_BUILD//+/p}
-      export NM=$($CC_FOR_BUILD -print-prog-name=nm)
-      export AR=$($CC_FOR_BUILD -print-prog-name=ar)
-      export OBJCOPY=$($CC_FOR_BUILD -print-prog-name=objcopy)
-      export STRIP=$($CC_FOR_BUILD -print-prog-name=strip)
+      _TOOLCHAIN_ARGS="CC=${CC_FOR_BUILD} CXX=${CXX_FOR_BUILD} CPP=${CXX_FOR_BUILD//+/p}"
```

Comment:
These can no longer be passed as env vars and should be passed on the cli directly instead